### ISSUE TITLE
Fix nsimg snippet (remove whitespace for stretch)

### ISSUE THREAD

**Issue by NickIliev** [tsvetan-ganev/nativescript-vscode-snippets#8]
Currently there is an additional whitespace for the stretch property when generating **nsimg**

it looks like this
```
<Image src="" stretch ="none" />
```

at it should be
```
 <Image src="" stretch="none" />
```

**Comment by lukeramsden** [comment]
Fixed in #9 (just needs to be merged)

**Comment by lukeramsden** [comment]
Solved by #9
